src/scratch/UCERF3/enumTreeBranches/ScalingRelationships.java

```diff
@@ -252,12 +252,69 @@ public double getArea(double mag, double origWidth) {
 			return map;
 		}
 
-	};
+	},
+	
+	TMG_SUB_2017("Thingbaijam et al.(2017) Subduction", "TMG_SUB_2017") {
+		// units of the input dimensions are in m or m^2 
+		public double getAveSlip(double area, double length, double origWidth) {
+			tmg_sub_magArea.setRake(90.0d); // interface
+			double mag = tmg_sub_magArea.getMedianMag(area*1e-6);
+			double moment = MagUtils.magToMoment(mag);
+			return FaultMomentCalc.getSlip(area, moment);
+		}
+		
+		public double getMag(double area, double origWidth) {
+			tmg_sub_magArea.setRake(90.0d); // interface
+			return tmg_sub_magArea.getMedianMag(area*1e-6);
+		}		
+		
+		public double getArea(double mag, double origWidth) {
+			tmg_sub_magArea.setRake(90.0d); // interface
+			return tmg_sub_magArea.getMedianArea(mag)*1e6;
+		}
+		
+		
+		@Override
+		public double getRelativeWeight(InversionModels im) {
+			// weight needs to be updated
+			return 1.0d; 
+
```

Comment:
there are some stray newlines in this file

Reply:
fixed